airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/S3DestinationTest.java

```diff
@@ -20,19 +34,66 @@
 public class S3DestinationTest {
 
   private AmazonS3 s3;
+  private AmazonS3 s3NoAccess;
```

Comment:
we can probably keep using a single s3 client, and instead add  `doThrow(..).when(s3)` block in the test itself. This makes it clearer what mocks the test itself is setting up and keeps the global namespace light

Reply:
done